src/cascadia/TerminalApp/JsonUtils.h

```diff
@@ -253,6 +262,11 @@ namespace TerminalApp::JsonUtils
             const auto string{ Detail::GetStringView(json) };
             return string.length() == 38 && string.front() == '{' && string.back() == '}';
         }
+
+        std::string TypeName() const
+        {
+            return "guid";

```

Comment:
Should we provide the format of the guid too? Especially considering that there's a few ways to represent a guid.

Reply:
we could, but it might make the messabe somewhat unwieldy